sql/core/src/test/scala/org/apache/spark/sql/test/TestSQLContext.scala

```diff
@@ -39,7 +39,10 @@ private[spark] class TestSparkSession(sc: SparkContext) extends SparkSession(sc)
 
   @transient
   override lazy val sessionState: SessionState = {
-    new TestSQLSessionStateBuilder(this, None).build()
+    val ss = new TestSQLSessionStateBuilder(this, None).build()
```

Comment:
`FakeDataSourceStrategy` is needed. otherwise::
```java
java.lang.AssertionError: assertion failed: No plan for Relation[a#23] FakeRelation(org.apache.spark.sql.SQLContext@6838be66)
```